DESCRIPTION

```diff
@@ -33,7 +35,9 @@ Suggests:
     ggplot2,
     janitor,
     lubridate,
+    testthat (>= 3.0.0),
```

Comment:
Can we move this to Suggests instead please? Think that's probably most appropriate for `testthat`.

Reply:
It is in Suggests? 😅

Reply:
ah sorry! I need to make sure to expand all the lines in the code review bit haha!